### PR TITLE
Fix/remove fuzzy

### PR DESCRIPTION
**Pull request by mohituniyal**
fixes https://github.com/openreview/openreview-py/issues/612

### CODE REVIEW

File: openreview/tools.py [hunk 1]
```diff
@@ -166,68 +161,6 @@ def create_profile(client, email, first, last, middle = None, allow_duplicates =
...
-            return [clean_name(n) for n in [first, last, middle]]
-        else:
-            return []
-
-    if 'authorids' in note.content and 'authors' in note.content:
-        author_names = [a.replace('*', '') for a in note.content['authors']]
-        author_emails = [e for e in note.content['authorids']]
-        if len(author_names) == len(author_emails):
-
-            email_by_authorname = match_authors_to_emails(author_names, author_emails)
```
Comment (melisabok) [inline openreview/tools.py:201]: can you remove this call and keep this function?

I think we should keep it. This function create profiles for current submissions.

File: tests/test_tools.py [hunk 2]
```diff
@@ -158,56 +158,6 @@ def test_parallel_exec(self, client):
...
-        shuffled_emails = random.sample(
-            [p[1] for p in correct_pairs], len(correct_pairs))
-
-        result = openreview.tools.match_authors_to_emails(
-            shuffled_names, shuffled_emails)
-
-        for name, email in correct_pairs:
-            assert result[name] == email
-
-    def test_create_authorid_profiles(self, client):
```
Comment (melisabok) [inline tests/test_tools.py:181]: keep this test please

File: tests/test_tools.py [hunk 3]
```diff
@@ -34,29 +34,19 @@ def test_add_members_to_group(self, client):
         posted_group = client.add_members_to_group(posted_group, ['test_subject_y1@mail.com', 'test_subject_y2@mail.com'])
         assert posted_group
         assert len(posted_group.members) == 3
-        assert 'test_subject_x@mail.com' in posted_group.members
-        assert 'test_subject_y1@mail.com' in posted_group.members
-        assert 'test_subject_y2@mail.com' in posted_group.members
+        assert all(list(map(lambda email: email in posted_group.members, ['test_subject_x@mail.com', 'test_subject_y1@mail.com', 'test_subject_y2@mail.com'])))
```
Comment (melisabok) [inline tests/test_tools.py:37]: is there a change in this PR making these tests fail? why are you changing them?

File: tests/test_tools.py [hunk 4]
```diff
@@ -228,10 +196,9 @@ def test_replace_members_with_ids(self, client, test_client):
             signatories=['~Super_User1'],
             members=['test@mail.com', '~Test_User1', '~Another_Name1']
         ))
-
         assert posted_group
 
-        client.post_profile(openreview.Profile(
+        _ = client.post_profile(openreview.Profile(
```
Comment (melisabok) [inline tests/test_tools.py:201]: what is this doing?

File: tests/test_tools.py (hunk 3 above)
Comment (melisabok) [reply tests/test_tools.py:37]: I prefer to keep the tests as simple as possible. Asserting each member individually is not a problem, it is clear for the users what the test is doing. 

If the test fails it is easier to know the reason. 

Also this is PR doesn't change any of this behavior, this change shouldn't be here.

File: tests/test_tools.py (hunk 2 above)
Comment (mohituniyal) [reply tests/test_tools.py:181]: Test has been restored already.

File: openreview/tools.py (hunk 1 above)
Comment (mohituniyal) [reply openreview/tools.py:201]: `get_names` has been restored.

File: tests/test_tools.py (hunk 3 above)
Comment (mohituniyal) [reply tests/test_tools.py]: Reverted to original asserts.

File: tests/test_tools.py (hunk 4 above)
Comment (mohituniyal) [reply tests/test_tools.py]: I was trying to follow the changes I was making in other places in this file. Anyway, I have reverted this too now.

**Pull request merged**